server/pairing/payload_manager.go

```diff
@@ -22,7 +24,10 @@ import (
 )
 
 var (
-	ErrKeyFileAlreadyExists = errors.New("key file already exists")
+	ErrKeyFileAlreadyExists       = errors.New("key file already exists")
+	ErrKeyUIDEmptyAsSender        = errors.New("keyUID must be provided as sender")
+	ErrNodeConfigNilAsReceiver    = errors.New("node config must be provided as receiver")
+	ErrPayloadSourceConfigBothSet = errors.New("payloadSourceSenderConfig and payloadSourceReceiverConfig cannot be both set")
```

Comment:
This is an interesting error. The fact that it warns it is possible to received both Sender and Receiver types is a sign that perhaps the config should be refactored. I'll add some notes below: